From bf99be994d6c8690cbb8446b8170f7723eaa7d33 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Wed, 17 Aug 2016 14:07:08 +0300
Subject: [PATCH 0681/2241] mmc: sdhci-xenon: Define the AP SD/eMMC clock
 connection

Add clock definition for SD/eMMC interface to APN806 DTSI.
This entry is required by the new Xenon driver.

Change-Id: Id022150feeb5a330b37856bd1809339ccee4c09a
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/32016
Reviewed-by: Omri Itach <omrii@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-ap806.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
index bae6d6e..cc4341b 100644
--- a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
@@ -285,6 +285,8 @@
 				compatible = "marvell,xenon-sdhci";
 				reg = <0x6E0000 0x300>;
 				interrupts = <GIC_SPI 16 IRQ_TYPE_LEVEL_HIGH>;
+				clock-names = "core";
+				clocks = <&syscon 4>;
 				xenon,slotno = <0>;
 				status = "disabled";
 			};
-- 
2.7.4

